Assets/MRTK/SDK/Profiles/DefaultMixedRealityControllerVisualizationProfile.asset

```diff
@@ -24,9 +24,9 @@ MonoBehaviour:
     type: 3}
   globalRightControllerModel: {fileID: 1760394390528354, guid: 9ef0df44319c4fb4a05649e5b8ceb6d1,
     type: 3}
-  globalLeftHandVisualizer: {fileID: 1227372834072826, guid: 378f91dbbff9e2343b27a467467750bf,
+  globalLeftHandVisualizer: {fileID: 61451597023024772, guid: 6080fb9e3b49fda449e3d70daf6d2254,
```

Comment:
@CDiaz-MS Before we change the rigged hand visualizer to the default, do you have time to verify its behavior using Leap Motion?

Reply:
Overall seems good! The joints don't align exactly with the model though.

![image](https://user-images.githubusercontent.com/3580640/104071857-dee33380-51be-11eb-9aba-49f2f83a4533.png)

@RogPodge Do you think there's anything specific that might be causing this?

Reply:
This is expected since everyone's hand size is a little bit different :( and there's no good way to adjust the mesh size based on where we think the joints are...

The current mesh is sized so that is looks pretty good when using the joint data we use for input simulation. In addition, when using the hand mesh on other platforms while hiding the joint data (ie Oculus Quest), the hand movement feels pretty accurate and natural from my experience @keveleigh.

I think I've set up the profiles so that the joint and the rigged mesh is only displayed together in editor, while other platforms use their own solutions (hololens should still be using the auto-generated hand mesh and Oculus should be displaying the rigged hand mesh without showing joint data.